docs/tutorials/getting_started/moran.rst

```diff
@@ -78,14 +77,33 @@ The scores in each round::
     [7.0, 7.0, 7.0, 0.0]
     [7.0, 7.0, 7.0, 0.0]
 
+We can plot the results of a Moran process with `mp.populations_plot()`. Let's
+use a larger population to get a bit more data::
+
+   >>> import random
+   >>> import matplotlib.pyplot as plt
+   >>> axl.seed(15)  # for reproducible example
+   >>> N = 10
+   >>> players = [random.choice(axl.demo_strategies)() for i in range(N)]
```

Comment:
Thanks for changing this, the plot file now needs to be updated to run with these strategies (eg `Suspicious Tit For Tat` is not in `demo_strategies`). Another alternative is to just hard code the names of the players in that plot file (instead of `random.choice(axl.demo_strategies)`) either way they need to match.